sklearn/cluster/hierarchical.py

```diff
@@ -733,6 +733,21 @@ class AgglomerativeClustering(BaseEstimator, ClusterMixin):
         at the i-th iteration, children[i][0] and children[i][1]
         are merged to form node `n_samples + i`
 
+    Examples
+    --------
+    >>> from sklearn.cluster import AgglomerativeClustering
+    >>> import numpy as np
+    >>> X = np.array([[1, 2], [1, 4], [1, 0],
+    ...               [4, 2], [4, 4], [4, 0]])
+    >>> clustering = AgglomerativeClustering().fit(X)
+    >>> clustering
+    ... # doctest: +NORMALIZE_WHITESPACE
```

Comment:
Maybe we can get rid of the `...` here? Current example seems strange from my side. See 
https://27375-843222-gh.circle-artifacts.com/0/doc/modules/generated/sklearn.cluster.AgglomerativeClustering.html

Reply:
that's strange, since according to `doctest`s documentation, it shouldn't be there. But trying to fix it now.

Reply:
should it be [WIP] until it's fixed now?

Reply:
Thanks. It's recommended to check the doc from Circle CI :)
Please apply the change to your other PRs.